src/test/java/com/egov/base/EGovTest.java

```diff
@@ -20,7 +20,7 @@
 import net.masterthought.cucumber.ReportBuilder;
 import net.minidev.json.JSONValue;
 
-@KarateOptions(features = {"classpath:com/egov/core-services","classpath:com/egov/business-services"}, tags = {"@Reports,@SearchMdms,@Searcher,@location,@localization,@userotp,@User,@accessControl,@hrms,@collectionServices"})
+@KarateOptions(features = {"classpath:com/egov/core-services","classpath:com/egov/business-services"}, tags = {"@Reports,@SearchMdms,@Searcher,@location,@localization,@userotp,@User,@accessControl,@hrms,@collectionServices,@billingServiceDemand"})
```

Comment:
Good to add EgovTest.java to .gitignore file as most of the times we are changing the tags and it will show file as modified. You can push only when it's change in the file other than tags. 
Let me know if this makes sens @krishnendu-eGov @yatheendra-egov ?